### PR TITLE
feat : darkmode

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR I'm adding dark mode to eCommerce

### CODE REVIEW

File: darkmode.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.dark-mode-toggle{
+    position: fixed;
+    right: 2rem;
+    bottom: 2rem;
+    transform: scale(.9);
+}
+
+.dark-mode-class{
+    background: #162538;
```
Comment (rutu14) [inline darkmode.css:9]: Color Variable

**Review by rutu14 — APPROVED** [review]
The dark mode looks great.

File: darkmode.css (hunk 1 above)
Comment (sudipkundu999) [reply darkmode.css:9]: I've intentionally not used color variable here as this color is not being used anywhere else.

File: darkmode.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.dark-mode-toggle{
```
Comment (zoef-yt) [inline darkmode.css:1]: Appreciation: The dark mode looks really great. Good job man.

**Review by zoef-yt — APPROVED** [review]
The UI looks great. Showing a little preview in the description would have been great. The code is good. Good to merge.

File: darkmode.css (hunk 2 above)
Comment (sudipkundu999) [reply darkmode.css:1]: Thank you! I'll keep the points about PR description in mind next time

**Pull request merged**